pages/how-to/set-up/scm-connection.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Setting Up an SCM Connection
+description: Connecting Plural to a Source Control Management Provider
+---
+
+# Prerequisites
+* **Plural Console `admin` Permissions**  
+* **SCM Provider Personal Access Token**
+  * [Github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)  
```

Comment:
For github we should strongly nudge people to configure it via a github application which is technically more secure and not bound to a specific user.